templates/site/index.html

```diff
@@ -0,0 +1,209 @@
+                {% set dropdown_label_text = "Share" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+        <div id="advancedFilters" class="div-advanced-filters">
+            <div>
+                <h6 class="title-advanced-filters">Advanced Filters</h6>
+            </div>
+
+            {% with %}
+                {% set dropdown_type = "dropdown_time_frame" %}
+                {% set dropdown_timeframe_id = "dropdownTimeframe" %}
+                {% set dropdown_timeframe_link_id = "dropdownTimeframeLabel" %}
+                {% set dropdown_timeframe_list_id = "dropdownTimeFrameList" %}
+                {% set user_datepicker_start_id  = "timeframeStartDatePicker" %}
+                {% set user_timepicker_start_id = "timeframeStartTimePicker" %}
+                {% set user_datepicker_end_id  = "timeframeEndDatePicker" %}
+                {% set user_timepicker_end_id = "timeframeEndTimePicker" %}
+                {% set user_checkbox_allday_id = "timeframeCheckboxAllDay" %}
+                {% set user_dropdown_repeat_id = "timeframeDropdownRepeat" %}
+                {% set user_dropdown_repeat_label_id = "timeframeDropdownRepeatLabel" %}
+                {% set user_datepicker_endson_id = "timeframeEndsOnDatepicker" %}
+                {% set dropdown_label_text = "Time Frame" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+            <div>
+                {% with %}
+                    {% set button_type = "button_location" %}
+                    {% set button_id = "showLocationMap" %}
+                    {% set button_label = "Location" %}
+                    {% include "partials/button.html" %}
+                {% endwith %}
+            </div>
+
+            {% with %}
+                {% include "partials/modal-map.html" %}
+            {% endwith %}
+
+            <div id="nonEnumQueryables" class="div-queryable-filters"></div>
+
+            <div id="dropdownQueryables" class="div-queryable-filters"></div>
+        </div>
+
+        </div>
+        <div class="content-search-right">
+            <div id="searchNavContainer" class="row div-search-nav-container">
+                <div class="content-search-title">
+                    <h5>Search <span id="titleSearchType" class="content-search-title-type"></span></h5>
+                </div>
+                <div class="content-search-description">
+                    <p class="body-1">
+                        Filter by date range to focus on specific timeframes,location to narrow down your search area,
+                        and keywords relevant to the type of imagery or dataset. Additional options, like cloud cover and
+                        sensor type, let you further customize your results to meet quality and relevance standards.
+                    </p>
+                </div>
+                <div id="searchFilterContainer" class="div-search-filter-container">
```

Comment:
This does not match the design. This should be in a box and the buttons should be formatted differently.